wslexe_windows.go

```diff
@@ -29,6 +29,58 @@ func terminate(distroName string) error {
 	return exec.Command("wsl.exe", "--terminate", distroName).Run()
 }
 
+// setAsDefault sets a particular distribution as the default one.
+//
+// It is analogous to
+//  `wsl.exe --set-default <distroName>`
+func setAsDefault(distroName string) error {
+	out, err := exec.Command("wsl.exe", "--set-default", distroName).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("error setting %q as default: %v, output: %s", distroName, err, out)
+	}
+	return nil
+}
+
+// defaultDistro gets the name of the default distribution.
+// If no distros are installed (hence no default), an empty string is returned.
```

Comment:
This is hard to exercise without breaking the testbed (since it would require unregistering all distros, which I'm not willing to do on my machine).

Reply:
understood :)